endpoints/openrtb2/auction.go

```diff
@@ -885,6 +885,23 @@ func setImpsImplicitly(httpReq *http.Request, imps []openrtb.Imp) {
 	}
 }
 
+func getJsonSyntaxError(testJSON []byte) (bool, string, []error) {
```

Comment:
Do you ever return anything besides `nil` for `[]error`?

Reply:
No.  Should I drop the third return value?